tests/test_date.py

```diff
@@ -2,66 +2,135 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from types import MethodType
+from itertools import izip
+from collections import OrderedDict
 import unittest
 from datetime import datetime, timedelta
 
 from dateutil.relativedelta import relativedelta
+
```

Comment:
You have `from` imports before `import` within single import block. Also, you should follow alphabetic order for the imports blocks (separately for `import` and `from ... import`)